src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -989,6 +992,19 @@ const Authenticate = (() => {
     };
 
     const onLoad = async () => {
+        const is_from_mt5 = Url.param('from_mt5');
+        if (!isAuthenticationAllowed() && !is_from_mt5) {
+            $('#authentication_tab').setVisibility(0);
+            $('#authentication_loading').setVisibility(0);
+
+            Dialog.alert({
```

Comment:
it shouldn't be a dialog, we can show the error message on the html page same as other errors